memgpt/agent_store/db.py

```diff
@@ -314,19 +314,13 @@ def get_all_cursor(
             # cursor logic: filter records based on before/after ID
             if after:
                 after_value = getattr(self.get(id=after), order_by)
-                if reverse:  # if reverse, then we want to get records that are less than the after_value
```

Comment:
We should probably keep `reverse` in, right? Was this just removed for debugging purposes?

Reply:
@goetzrobin I reverted the change on this file - I'm assuming that the diff isn't actually needed in this PR?